### PR TITLE
fix: removed extra padding around fast card component

### LINKED ISSUE #3440: Border padding being added in fast-card
Trying to use fast-card to create a new card type.
Seeing a small amount of padding being added around the border of the card.

For my design requirements I need my image content to extend all the way to the border edge.
Can this default styling be changed or overwritten?

image
image

### PR DESCRIPTION
**Pull request by khamudom**
# Description

There was a transparent border set around the card that is only visible in high contrast.
High contrast work will be done in a separate PR

## Motivation & context

closes #3440 

before
image
after
image

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/web-components/fast-components-msft/src/card/card.styles.ts [hunk 1]
```diff
@@ -13,6 +14,16 @@ export const CardStyles = css`
...
+    neutralLayerCardBehavior,
+    forcedColorsStylesheetBehavior(
+        css`
+            :host {
+                forced-color-adjust: none;
+                background: ${SystemColors.Canvas};
+                box-shadow: 0 0 0 1px ${SystemColors.CanvasText};
+            }
+        `
+    )
```
Comment (chrisdholt) [inline packages/web-components/fast-components-msft/src/card/card.styles.ts:28]: This looks like a feature - should we call out that you're including high contrast styles here?

Comment (chrisdholt) [reply packages/web-components/fast-components-msft/src/card/card.styles.ts:28]: Should these be seperate PR's? Once appears to be a fix, the other appears to be a feature.

Comment (khamudom) [reply packages/web-components/fast-components-msft/src/card/card.styles.ts:28]: I put them in one as to not regress the high contrast work when fixing the padding bug. But I am all for making them separate PRs.

File: packages/web-components/fast-components-msft/src/card/card.styles.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { css } from "@microsoft/fast-element";
-import { display } from "@microsoft/fast-foundation";
+import { display, forcedColorsStylesheetBehavior } from "@microsoft/fast-foundation";
 import { elevation } from "../styles";
 import { neutralLayerCardBehavior } from "../styles/index";
+import { SystemColors } from "@microsoft/fast-web-utilities";
```
Comment (chrisdholt) [inline packages/web-components/fast-components-msft/src/card/card.styles.ts:5]: @khamudom I know these are coming in with the other PR, but they should be removed from here as they are currently referenced.

Comment (khamudom) [reply packages/web-components/fast-components-msft/src/card/card.styles.ts:5]: shoot. thank you!

**Pull request merged**